CreeDictionary/API/search/runner.py

```diff
@@ -26,4 +37,7 @@ def search(*, query: str, include_affixes=True, include_auto_definitions=False):
         do_source_language_affix_search(search_run)
         do_target_language_affix_search(search_run)
 
+    if cvd_search_type != CvdSearchType.OFF:
+        do_cvd_search(search_run)
```

Comment:
Is there a truthy test that could be done here instead? As in `cvd_search_type == blah` instead of `!=`?
It's a small thing, I just find truthy statements easier to read.